workflow/scripts/variant_filtration.py

```diff
@@ -1,5 +1,4 @@
 from pysam import VariantFile
```

Comment:
it would be nice to have units tests for the script

Reply:
Do you know any valuable resources or tutorials about this subject? I don't have any previous experience with unit testing.

Reply:
Skip it for now, but add an issue remanding that unit tests should be created.

I will see if I can find some good tutorials.

Reply:
Sounds good, thank you very much! I have added the task to GitHub Projects.